common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -184,7 +185,7 @@ public static boolean tryFociTokenWithGivenClientId(@SuppressWarnings(WarningTyp
         final MicrosoftStsOAuth2Strategy strategy = new MicrosoftStsOAuth2Strategy(config, strategyParameters);
 
         final String refreshToken = refreshTokenRecord.getSecret();
-        final String scopes = BaseController.getDelimitedDefaultScopeString();
+        final String scopes = AuthenticationConstants.OAuth2Scopes.OPEN_ID_SCOPE;
```

Comment:
Why changing this?

Reply:
This seems like a mistake.  What should this be?

Reply:
I was trying to scope it down to the minimum - so that the application that does not require MSGraph does not have to request preauthorization for the other 2 - which is optional in the form.

Maybe it's not a good idea.

Reply:
Can we send the customary `offline_access openid profile` and not specify a resource? Seems preferable, as MS Graph doesn't exist in all clouds

Reply:
https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes

The Microsoft identity platform implementation of OpenID Connect has a few well-defined scopes _**that are also hosted on the Microsoft Graph**_: openid, email, profile, and offline_access. The address and phone OpenID Connect scopes are not supported.

This describes why I was hitting the MSgraph preauthorized error with those scopes.

Reply:
https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes

The Microsoft identity platform implementation of OpenID Connect has a few well-defined scopes _**that are also hosted on the Microsoft Graph**_: openid, email, profile, and offline_access. The address and phone OpenID Connect scopes are not supported.

This describes why I was hitting the MSgraph preauthorized error with those scopes (AADSTS65002: Consent between first party application '[CLIENT_ID]' and first party resource '00000003-0000-0000-c000-000000000000' must be configured via preauthorization. Visit https://identitydocs.azurewebsites.net/static/aad/preauthorization.html).